test/rules/first.spec.ts

```diff
@@ -25,7 +25,7 @@ ruleTester.run('first', rule, {
             import { x } from 'foo';",
     }),
     test({
-      // issue #2210
+      name: '...component.html (issue #2210)',
```

Comment:
Cleans up the test summary since this file is ~150 lines long

Reply:
Nice!